cmd/chartmuseum/main.go

```diff
@@ -76,10 +88,8 @@ func cliHandler(c *cli.Context) {
 		ChartPostFormFieldName: conf.GetString("chartpostformfieldname"),
 		ProvPostFormFieldName:  conf.GetString("provpostformfieldname"),
 		ContextPath:            conf.GetString("contextpath"),
-		LogJSON:                conf.GetBool("logjson"),
```

Comment:
If LogJSON and Debug fields are no longer effective can you add a comment in ServerOptions prefixed with `// Deprecated: <reason>`?

Reply:
definitely, added in d84441b9718a08b412716104fb03fe9d71b8fb3a!